components/Masthead.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+* Masthead.tsx
+* @2023 Open Seattle
+*/
+
+import { Stack, Typography, styled, useMediaQuery, useTheme } from "@mui/material"
+
+type MastheadProps = {
```

Comment:
This is nice. We should consider using this whenever we have title that does not have image then!

Reply:
I'll do that after this is merged into rebrand-2023